ctapipe/image/tests/test_extractor.py

```diff
@@ -348,6 +348,22 @@ def test_two_pass_window_sum(subarray):
         assert_allclose(charge, true_charge, rtol=0.1)
         assert_allclose(pulse_time, true_time, rtol=0.1)
 
+    # test only 1st pass
+    extractor.disable_second_pass = False
```

Comment:
For only the first pass, `disable_second_pass=True`, right?

Reply:
yes I should have fixed it few minutes ago

Reply:
oh god actually it was correct before ahah
now I get what you mean